app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -75,8 +78,21 @@ const AppLayoutWithSidebarAndData = ({
         </h2>
         {children({ experiment })}
       </section>
-    </AppLayoutWithSidebar>
+    </Layout>
   );
 };
 
-export default AppLayoutWithSidebarAndData;
+const Layout = ({
+  hasSidebar,
```

Comment:
This is purely a personal preference so feel free to disregard, but `sidebar` feels good here. A little less verbose.

Reply:
Hmm yeah the word `has` always implies a boolean to me. I might think "which sidebar" with `sidebar` (enum or maybe a React Element) but I suppose that's what Typescript is for. ;)